File: mario.py
height = int(input('please pick a number between 1 and 8 ' ))
while not 1<= height<= 8:
    height = int(input('please pick a number between 1 and 8 '))

for i in range(1, height + 1):
    for j in range(1 , height + 1):
        if i <= height -j:
            print(' ', end='')
        else:
            print('#', end='')
    print('\n')

File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Please enter a number between 1 and 8 ");
    }
    while (height < 1 || height > 8);

    for (int i = 1; i < height + 1; i++)
    {
        for (int j = 1; j < height + 1; j++)
        {
            if (i <= height - j)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("  ");
        for (int k = 1; k < height + 1; k++)
        {
            if (k > height - i)
            {
                printf("#");
            }
        }
        printf("\n");
    }
}
